cmonge/trainers/ot_trainer.py

```diff
@@ -153,6 +155,7 @@ def setup(
         }
 
         # Save for use in `evaluate`
+        # Potentially overwrite AbstractTrainer attribute
```

Comment:
This is not necessary, just move the attribute to the constructor of this class and then pass it down to the parent class via `super().__init__()`